ui/src/features/Editor/components/OverlayUI/components/DebugPanel/DebugPreview/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { GlobeIcon, MapPinAreaIcon, WarningIcon } from "@phosphor-icons/react";
+import { memo } from "react";
+
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogContentSection,
+  DialogContentWrapper,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  LoadingSkeleton,
+  Tabs,
+  TabsContent,
+  TabsList,
+  TabsTrigger,
+} from "@flow/components";
+import { useT } from "@flow/lib/i18n";
+import { JobState } from "@flow/stores";
+import type { SupportedDataTypes } from "@flow/utils/fetchAndReadGeoData";
+
+import { ThreeDViewer, TwoDViewer } from "./components";
+
+type Props = {
+  fileType: SupportedDataTypes | null;
+  selectedOutputData: any;
+  debugJobState?: JobState;
+  isLoadingData: boolean;
+  dataURLs?: { key: string; name: string }[];
+  showTempPossibleIssuesDialog: boolean;
+  onShowTempPossibleIssuesDialogClose: () => void;
+};
+const DebugPreview: React.FC<Props> = ({
+  fileType,
+  debugJobState,
+  selectedOutputData,
+  dataURLs,
+  isLoadingData,
+  showTempPossibleIssuesDialog,
+  onShowTempPossibleIssuesDialogClose,
+}) => {
+  const t = useT();
+
+  return debugJobState && dataURLs ? (
+    <Tabs
+      className={`pointer-events-auto h-full w-full rounded-md bg-secondary p-1 shadow-md shadow-secondary transition-all`}
+      defaultValue="2d-viewer">
+      <div className="relative flex items-center">
+        <div className="flex w-full items-center p-1">
+          <TabsList className="flex w-full justify-between">
```

Comment:
Please review the CSS. Divs with only one child don't need to be flexed or have items centered etc. Also a random relative class, etc. 
If you play around with CSS to get things working right, always do a post-step of removing stuff that seems not needed and reducing down to the least amount of CSS. Do the same for elements too.